docs/features/message-system.md

```diff
@@ -269,6 +264,40 @@ Structure of config, types and optionality of specific keys can be found in the
                 ]
             }
         }
+    ],
+    // Used for AB testing
+    "experiments": [
+        {
+            // Same as for actions, check conditions above
+            "conditions": [
+                {
+                    "environment": {
+                        "desktop": ">=24.5.1",
+                        "mobile": "!",
+                        "web": ">=24.5.1"
+                    }
+                }
+            ],
+            // Detail of an experiment
+            "experiment": {
+                // Used as a selector for the test
```

Comment:
There was a request to make it human-readable, but it will require more control from the person who will add it.

Reply:
For the map I mean simple constant somewhere in the codebase that can be used instead of the uuid like 

```javascript
const experiments = {
 orangeSendButton: 'fb0eb1bc-8ec3-44d4-98eb-53301d73d981',
 somethingElse: 'def30808-589b-4aab-92a2-334b2104e6c4',
} as const;
```

I think that is a no-brainer and we should translate those random ids to something readable and not put the strings everywhere. We can even keep the old experiments there so we will never reuse the same id.

Reply:
I have added the mapping here [13573a0](https://github.com/trezor/trezor-suite/pull/15281/commits/13573a092dbbe5a4acd067f84ffc096133a18171) and also created relatively strong typing.